tests/Monolog/Handler/ElasticaHandlerTest.php

```diff
@@ -226,13 +281,18 @@ protected function getCreatedDocId(Response $response)
      * Retrieve document by id from Elasticsearch
      * @param  Client $client     Elastica client
      * @param  string $index
-     * @param  string $type
+     * @param  ?string $type
      * @param  string $documentId
      * @return array
      */
     protected function getDocSourceFromElastic(Client $client, $index, $type, $documentId)
     {
-        $resp = $client->request("/{$index}/{$type}/{$documentId}", Request::GET);
+        if($type === null) {
+            $path  = "/{$index}/{$documentId}";
```

Comment:
the new API endpoint to get a document is `/index_name/_doc/document_id`

Reply:
Fixed in https://github.com/Seldaek/monolog/pull/1523/commits/79d12135dd08837f5a595d9699fa56a5303f1c05